### PR TITLE
Add parsing of altitude value

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Selenium API also requires `altitude` value to set. The PR depends on https://github.com/appium/io.appium.settings/pull/17

### CODE REVIEW

File: lib/tools/adb-commands.js [hunk 1]
```diff
@@ -913,13 +915,14 @@ methods.getGeoLocation = async function () {
       `services must be enabled on the device. Original error: ${err.message}`);
   }
 
-  const match = /data="(-?[\d\.]+)\s+(-?[\d\.]+)"/.exec(output);
+  const match = /data="(-?[\d\.]+)\s+(-?[\d\.]+)\s+(-?[\d\.]+)"/.exec(output);
```
Comment (jlipps) [inline lib/tools/adb-commands.js]: does this regex allow for the third number to be absent in the case of running an older appium settings?

Comment (mykola-mokhnach) [reply lib/tools/adb-commands.js]: no it does not, but I assume there will by anyway the minimum package version set in android driver, so we don't have to care much about it

Comment (jlipps) [reply lib/tools/adb-commands.js]: ok, as long as android driver performs an upgrade on the settings app if necessary 👍

Comment (mykola-mokhnach) [reply lib/tools/adb-commands.js]: yes it does
https://github.com/appium/appium-android-driver/blob/05e7a982f4c44c7bd09922caf02079d3e561681c/lib/android-helpers.js#L438

**Pull request merged**